go.mod

```diff
@@ -94,9 +95,8 @@ require (
 	github.com/pelletier/go-toml v1.9.1 // indirect
 	github.com/pierrec/lz4/v4 v4.1.7 // indirect
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.10.0
+	github.com/prometheus/client_golang v1.11.0
```

Comment:
All of these go.mod updates seem unrelated. Is that okay?

Reply:
Hmm, I didn't noticed that, it shouldn't include dependency changes.